checkov/cloudformation/cfn_utils.py

```diff
@@ -103,12 +103,7 @@ def get_folder_definitions(
             if file_ending in CF_POSSIBLE_ENDINGS:
                 files_list.append(os.path.join(root, file))
 
-    definitions, definitions_raw = get_files_definitions(
-        files_list, out_parsing_errors, lambda f: f'/{os.path.relpath(f, os.path.commonprefix((root_folder, f)))}')
-
-    definitions = {create_file_abs_path(root_folder, file_path): v for (file_path, v) in definitions.items()}
-    definitions_raw = {create_file_abs_path(root_folder, file_path): v for (file_path, v) in definitions_raw.items()}
```

Comment:
why are those 2 lines removed?

Reply:
I saw we change the file path when calling get_files_definitions function and than change it back, so I removed the two changes.